rest-client/src/main/scala/com/dota/rest/client/Constants.scala

```diff
@@ -0,0 +1,17 @@
+package com.dota.rest.client
+
+/**
+  * Constants that are used in applicaiton that are not configurable
+  */
+object Constants {
```

Comment:
maybe rename to RestUrlConstants

Reply:
I thought all other constants that we will need, we be also there